src/sys/unix.rs

```diff
@@ -1479,6 +1501,28 @@ impl crate::Socket {
         }
     }
 
+    /// Sets `SO_PEERCRED` to null on the socket.
+    /// This is a Cygwin extension.
```

Comment:
Please view how things get rendered in the html docs. You need an empty line to get a newline.
```suggestion
    /// Sets `SO_PEERCRED` to null on the socket.
    ///
    /// This is a Cygwin extension.
```